apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobReadCarrierWithTaxNumbers.json

```diff
@@ -227,9 +227,8 @@
             "display" : "National Provider Identifier"
           } ]
         },
-        "value" : "1497758544"
-      },
-      "display" : "CUMBERLAND COUNTY HOSPITAL SYSTEM, INC"
```

Comment:
why did we lose this?

Reply:
we added a different value

Reply:
this seems like a backward incompatibility